src/test/java/uk/gov/pay/api/model/PaymentTest.java

```diff
@@ -46,8 +46,9 @@ public void shouldNotPrintEmailAndCardDetailsWhenToString() throws IOException {
                 "    \"finished\": false\n" +
                 "  }\n" +
                 "}", ChargeFromResponse.class);
+        Charge charge = Charge.from(paymentFromConnector);
 
-        PaymentWithAllLinks payment = PaymentWithAllLinks.valueOf(paymentFromConnector, selfUri, eventsUri, cancelUri, refundsUri, captureUri);
+        PaymentWithAllLinks payment = PaymentWithAllLinks.valueOf(charge, selfUri, eventsUri, cancelUri, refundsUri, captureUri);
```

Comment:
I have no idea what the payment - charge distinction is in this codebase. Maybe something to think about as you make your changes. If there is no distinction we should settle on one term - probably payment

Reply:
Can change it - just wanted the code to work in the first pass ;)